exp-player/addon/components/exp-page-number/component.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
```

Comment:
Small request: could you rename this component?

The `exp-` prefix has a very special meaning in Experimenter: it means that something can be used as a frame of an experiment. Other utility components, like radio-group, do not use this prefix for that reason.

https://github.com/abought/exp-addons/blob/ece08446e342d9cc888123fa897fdeaed35e89ef/exp-player/addon/utils/parse-experiment.js#L79-L79